src/neo/SmartContract/Native/Tokens/NeoToken.Economic.cs

```diff
@@ -0,0 +1,178 @@
+using Neo.Cryptography.ECC;
+using Neo.IO;
+using Neo.Ledger;
+using Neo.Persistence;
+using System;
+using System.Numerics;
+
+namespace Neo.SmartContract.Native.Tokens
+{
+    public partial class NeoToken
+    {
+        private const byte Prefix_GasPerBlock = 17;
+        private const byte Prefix_NeoHoldersRewardRatio = 73;
+        private const byte Prefix_CommitteeRewardRatio = 19;
```

Comment:
I think that we are getting a lot of `Prefix`. I already mentioned that in a last PR, perhaps now we need a common global sharing of these values.

Reply:
It's good for me, I can update it.

Reply:
These prefixes are for this contract only, why make them become global?

Reply:
We need a mexanism for simplidy overlap of them. They are used local but values can not be used twice in different scopes,right?

Reply:
> They are used locally but values can not be used twice in different scopes,right?

Yes, they can.